src/components/rockets/ReserveButton.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './ReserveButton.css';
+
+const ReservedButton = () => (
+  <button className="rButton" type="button">Reserve Rocket</button>
```

Comment:
Maybe here is better to used `-` for the `className="r-button"`.

Reply:
OK, I see that in the rest of the files you have done this, but my understanding was that in CSS class names are `x-y`. This is not JS. But if you keep them consistent you may ignore this comment.